src/components/form/input.js

```diff
@@ -22,7 +22,7 @@ const InputWrapper = styled.div`
         border-color: var(--color-grey-5);
 
         & > label {
-            color: var(--color- ${(props) => props.labelHoverColor || 'black-3'});
+            color: var(--color-${(props) => props.labelHoverColor || 'black-3'});
```

Comment:
You can add `   /* prettier-ignore */` above this line to prevent creating an extra space.